### DISCUSSION TITLE
posts/markdown-syntax/

### DISCUSSION THREAD

**Discussion by giscus** [way2ml/comments#115] [General]
# posts/markdown-syntax/

Sample article showcasing basic Markdown syntax and formatting for HTML elements.

https://way2ml.github.io/posts/markdown-syntax/

**Comment by way2ml** [comment]
This is a test.

**Reply by huangchieh** [reply]
It works really well. @way2ml

**Reply by HuangJiaLian** [reply]
I'm very glad to see the new design.  It looks elegant. @huangchieh

**Reply by HuangJiaLian** [reply]
I am also curious about whether formula can be render correctly. Let try. First inline equation: $a^2 + b^2 = c^2$. Then separated equation. 

$$
E = mc^2
$$

**Reply by HuangJiaLian** [reply]
What about image.  Looks wonderful. 
![](https://cdn.jsdelivr.net/gh/HuangJiaLian/DataBase0@master/uPic/2021_12_16_21_SteadyHand.jpeg)
